dist/init/linux-systemd/caddy.service

```diff
@@ -33,16 +38,25 @@ PrivateDevices=true
 ProtectHome=true
 ; Make /usr, /boot, /etc and possibly some more folders read-only.
 ProtectSystem=full
-; … except /etc/ssl/caddy, because we want Letsencrypt-certificates there.
+; … except /etc/ssl/caddy, because we want Letsencrypt-certificates there
+;   and /var/log/caddy, because we want a place where logs can go.
 ;   This merely retains r/w access rights, it does not add any new. Must still be writable on the host!
-ReadWriteDirectories=/etc/ssl/caddy
+ReadWritePaths=/etc/ssl/caddy /var/log/caddy
 
 ; The following additional security directives only work with systemd v229 or later.
 ; They further retrict privileges that can be gained by caddy. Uncomment if you like.
 ; Note that you may have to add capabilities required by any plugins in use.
-;CapabilityBoundingSet=CAP_NET_BIND_SERVICE
-;AmbientCapabilities=CAP_NET_BIND_SERVICE
-;NoNewPrivileges=true
+CapabilityBoundingSet=CAP_NET_BIND_SERVICE
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+NoNewPrivileges=true
+
+; Caveat: Some plugins need additional capabilities. Add them to both lines above.
+; - plugin "upload" needs: CAP_LEASE
 
 [Install]
 WantedBy=multi-user.target
+
+; Learn more:
+;
+;   * systemd: https://www.freedesktop.org/software/systemd/man/systemd.exec.html
```

Comment:
I don't think this is the best place to start getting familiar with **systemd**. :-)